GuiltMotivator/app/src/main/java/nielsen/guiltmotivator/EmailService.java

```diff
@@ -131,30 +131,36 @@ private void sendEmail(Context context, Task task){
         for (int i = 0; i < contacts.size();i++){
             //Creating SendMail object
             String msg = getMessage(right_pronoun_objective, right_pronoun_subjective,
-                    right_pronoun_possessive, right_pronoun_subj_article, tone, name, contacts.get(i).getName(), task.getText());
+                    right_pronoun_possessive, right_pronoun_subj_article, contacts.get(i).getTone(), name, contacts.get(i).getName(), task.getText(), task.isChecked());
```

Comment:
yep. not implemented rn, but it's so that it's possible to send positive feedback.